test/indexing.jl

```diff
@@ -20,6 +20,23 @@
         end
     end
 
+    @testset "Searching within a nontrivial directory" begin
+        # https://github.com/invenia/Checkpoints.jl/issues/39
+        mktempdir() do outer_path
+            # This path is tricky, it is more than 1 folder deep
+            # and it has `=` in bits that are not tags
+            path = mkdir(joinpath(mktempdir(Path),"a","b=1","c"); recursive=true)
```

Comment:
`outer_path` was unused, did you mean

```suggestion
        mktempdir() do outer_path
            # This path is tricky, it is more than 1 folder deep
            # and it has `=` in bits that are not tags
            path = mkdir(joinpath(outer_path,"a","b=1","c"); recursive=true)
```

Note I didn't find a `mktempdir(Path)` method that allows `do` block so just using String.